File: Selenium/Project/LMS_Activity5.java
package project;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.openqa.selenium.support.ui.ExpectedConditions;

public class LMS_Activity5 {

	public static void main(String[] args) {

		System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");

		WebDriver driver = new ChromeDriver();

		try {

			driver.get("https://alchemy.hguy.co/lms");

			WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(20));
			wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("nav")));

			WebElement myAccountMenuItem = driver.findElement(By.linkText("My Account"));
			myAccountMenuItem.click();

			wait.until(ExpectedConditions.titleContains("My Account"));

			String pageTitle = driver.getTitle();

			if ("My Account – Alchemy LMS".equals(pageTitle)) {
				System.out.println("Successfully navigated to the My Account page.");
			} else {
				System.out.println("Failed to navigate to the correct page. The current title is: " + pageTitle);
			}
		} catch (Exception e) {
			e.printStackTrace();
		} finally {

			driver.quit();
		}
	}
}

File: Selenium/Project/lms_Activity5.py
#Goal: Go to "My Account” link in the page and verify the page title
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

    # Start the Driver
with webdriver.Firefox() as driver:

    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms/")
    driver.maximize_window()

    #Find the menu item that says “My Account” and click it
    driver.find_element(By.XPATH, "//ul[contains(@class,'nav-')]//a[contains(text(),'My Account')]").click()
    myaccount_page_title = driver.title.replace("–", "")
    print(myaccount_page_title)

    # Get  the title of the page and verify it
    if(myaccount_page_title == "My Account  Alchemy LMS"):
        print ("The Page title is : ",driver.title)
    else:
         print ("Wrong Website ") 

    #close the browser
    driver.quit()